### PR TITLE
add new curators (very sorry for the delay)

### LINKED ISSUE #100: Forum Helpers List: add curators
A checkbox indicates a user has provided a bio.

**Describe the changes needed**
add the following people:

- [x] ninjaMAR
- [x] Futurebot5
- [x] Steve0Greatness
- [ ] ScratchCatHELLO
- [x] mybearworld
- [x] Jeffalo
- [x] dogsmakemehappy
- [x] -EmeraldThunder-
- [x] scratchykit5743
- [x] 9gr
- [ ]  Nezon

**Additional context**
ill do it soon

NOTE: If you are a user on this list please do provide a bio by replying to this issue.

Copy and paste message:

Hello! Congrats on getting accepted to TFH! If you don't mind, could you please provide a bio for our website? You can see some examples here: https://theforumhelpers.github.io/forumhelpers/ Please comment the bio you would like as well as your Scratch user id (find it at api.scratch.mit.edu/users/YOURUSERNAME/) on the following GitHub issue: https://github.com/theforumhelpers/theforumhelpers.github.io/issues/100 If you absolutely can't comment on GitHub, you may reply to this comment. Thanks!

👀  =  i viewed

yo 100 issues/prs

### PR DESCRIPTION
**Pull request by leahcimto**
Adds new curators that have joined the studio.

### Resolves:
Resolves #100 

### Changes:
adds new curators

### Local Tests:
none yet though I'll test soon

again sorry that this took so long.

**Really important notes:**

- Not sure jeffalo's bio would be ok as it might go against the userscript policy even though that extension is completely safe
- that's it

@Accio1 

(note to self, test locally soon)

### CODE REVIEW

File: forumhelpers/curators.json [hunk 1]
```diff
@@ -425,7 +425,118 @@
...
+
+  {
+        "name": "ninjaMAR",
+        "id": 35527172,
+        "bio": "I'm ninjaMAR, a coder who mainly uses python. I am often found it the Advanced Topics helping people. I enjoy experimenting and I also have 1000+ posts",
+        "helped_code": false
+  },
+    
+  {
+        "name": "Arqwa",
```
Comment (Accio1) [inline forumhelpers/curators.json:438]: It looks like Arqwa is in here twice

**Review by Accio1 — CHANGES_REQUESTED** [review]

**Comment by Accio1** [comment]
jeffalo's biography should be fine. It doesn't link to the Scratch Addons site or go into much detail about it, so I think that it's okay to leave on the site.

**Comment by Accio1** [comment]
Also, despite the .gitignore file, this commit is still committing a .DS_Store file. I don't really know why.

**Comment by leahcimto** [comment]
> Also, despite the .gitignore file, this commit is still committing a .DS_Store file. I don't really know why.

well that's irritating

**Review by Accio1 — APPROVED** [review]
Looks good!

**Pull request merged**